src/GDGUkraine/rest_controller.py

```diff
@@ -63,6 +65,69 @@ def info(self):
                     req.admin_user['filter_place']))
         return res
 
+    @cherrypy.tools.json_out()
+    def sign_in(self):
+        # Doc:
+        # https://developers.google.com/identity/sign-in/web/backend-auth
+        # #using-a-google-api-client-library
+
+        req = cherrypy.request
+
+        try:
+            # id_token = req.json['id_token']
+            access_code = req.json['access_code']
+            ##
```

Comment:
Consider either deleting commented code or moving it to another feature branch, where it will be used. Keep the repository and code clean.